console-subscriber/examples/grpc_web.rs

```diff
@@ -0,0 +1,122 @@
+//! Example of using the console subscriber with tonic-web.
+//! This example requires the `grpc-web` feature to be enabled.
+//! Run with:
+//! ```sh
+//! cargo run --example grpc_web --features grpc-web
+//! ```
+use std::{thread, time::Duration};
+
+use console_subscriber::{ConsoleLayer, ServerParts};
+use http::header::HeaderName;
+use tonic_web::GrpcWebLayer;
+use tower_http::cors::{AllowOrigin, CorsLayer};
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+const DEFAULT_MAX_AGE: Duration = Duration::from_secs(24 * 60 * 60);
+const DEFAULT_EXPOSED_HEADERS: [&str; 3] =
+    ["grpc-status", "grpc-message", "grpc-status-details-bin"];
+const DEFAULT_ALLOW_HEADERS: [&str; 5] = [
+    "x-grpc-web",
+    "content-type",
+    "x-user-agent",
+    "grpc-timeout",
+    "user-agent",
+];
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    let (console_layer, server) = ConsoleLayer::builder().with_default_env().build();
+    thread::Builder::new()
+        .name("subscriber".into())
+        .spawn(move || {
+            // Do not trace anything in this thread.
+            let _subscriber_guard;
+            _subscriber_guard =
+                tracing::subscriber::set_default(tracing_core::subscriber::NoSubscriber::default());
```

Comment:
nit: this can be put in a single statement in this case.

```suggestion
            let _subscriber_guard =
                tracing::subscriber::set_default(tracing_core::subscriber::NoSubscriber::default());
```

Reply:
Updated! Good catch.